index.js

```diff
@@ -10,7 +10,7 @@ const assumeRole = async config => {
 	const AWS = require('aws-sdk'); // eslint-disable-line global-require
 	AWS.config.region = config.provider.region;
 	const sts = new AWS.STS({ apiVersion: '2011-06-15' });
-
+	console.log(`Attempting to assume AWS role ${config.provider.role}`);
 	return sts
 		.assumeRole({
 			RoleArn: config.provider.role,
```

Comment:
One of the deprecation notices for Serverless v3 is to move the role into an iam property (see screenshot). What's your opinion on how to manage this? We should probably support both possibilities.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/87309844/185611905-e31e0264-0ffe-47c9-93f4-5657fec3b487.png">